files/__addonLocation__/rollup.config.__rollupExt__

```diff
@@ -14,14 +14,20 @@ export default {
   plugins: [
     // These are the modules that users should be able to import from your
     // addon. Anything not listed here may get optimized away.
-    addon.publicEntrypoints(['components/**/*.js', 'index.js']),
```

Comment:
Yup, aware of it! :+1:

Reply:
yeah, it needs `@embroider/addon-dev@^2.0.0`, and then it should work

Reply:
Oh, I see. I could have checked that myself 🤦‍♂️ . I thought we were using the latest already here, but because of the major bump and us still referencing `^1.0.0` we weren't. Can confirm it is working now! 🙏